.github/workflows/gh-release.yml

```diff
@@ -1,5 +1,7 @@
 on:
   push:
+    branches: # specify branch to Mitiq master not forks
+      - master
```

Comment:
This should be the 'release' branch if I understand correctly

Reply:
I guess I was confused about how `release` branch is created. I thought this workflow was going to create a `release` branch :sweat_smile: !